apps/payments/next/middleware.ts

```diff
@@ -2,8 +2,23 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 import { NextRequest, NextResponse } from 'next/server';
+import { supportedLanguages } from '@fxa/shared/l10n';
+
+const DEFAULT_LOCALE = 'en-US';
 
 export function middleware(request: NextRequest) {
+  // Handle locale fallback
+  const pathname = request.nextUrl.pathname;
+  const segments = pathname.split('/');
+  const localeSegment = segments[1];
```

Comment:
nit. this can be reduced to a single line. `const localeSegment = request.nextUrl.pathname.split('/')[1];`